### PR TITLE
Add a new keyboard, the N87

### PR DESCRIPTION
**Pull request by NightlyBoards**
Adding a new keyboard, the N87. A TKL PCB that supports standard, tsangan, and symmetric standard bottom row

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/nightly_boards/n87/config.h [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+#define LCD_DATA3_PIN    7            //< pin for 4bit data bit 3
+#define LCD_RS_PORT      LCD_PORT     //< port for RS line
+#define LCD_RS_PIN       3            //< pin  for RS line
+#define LCD_RW_PORT      LCD_PORT     //< port for RW line
+#define LCD_RW_PIN       2            //< pin  for RW line
+#define LCD_E_PORT       LCD_PORT     //< port for Enable line
+#define LCD_E_PIN        1            //< pin  for Enable line
+#endif
+*/
+
```
Comment (fauxpark) [inline keyboards/nightly_boards/n87/config.h:248]:
```suggestion
```

File: keyboards/nightly_boards/n87/info.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        }
+	}
+}
+
```
Comment (fauxpark) [inline keyboards/nightly_boards/n87/info.json:19]:
```suggestion
{
    "keyboard_name": "[n]87",
    "url": "",
    "maintainer": "Neil Brian Ramirez",
    "width": 18.25,
    "height": 6.25,
    "layouts": {
        "LAYOUT_tkl_ansi_split_bs_rshift": {
            "layout": [
                {"x": 0, "y": 0},
                {"x": 2, "y": 0},
                {"x": 3, "y": 0},
                {"x": 4, "y": 0},
                {"x": 5, "y": 0},
                {"x": 6.5, "y": 0},
                {"x": 7.5, "y": 0},
                {"x": 8.5, "y": 0},
                {"x": 9.5, "y": 0},
                {"x": 11, "y": 0},
                {"x": 12, "y": 0},
                {"x": 13, "y": 0},
                {"x": 14, "y": 0},
                {"x": 15.25, "y": 0},
                {"x": 16.25, "y": 0},
                {"x": 17.25, "y": 0},

                {"x": 0, "y": 1.25},
                {"x": 1, "y": 1.25},
                {"x": 2, "y": 1.25},
                {"x": 3, "y": 1.25},
                {"x": 4, "y": 1.25},
                {"x": 5, "y": 1.25},
                {"x": 6, "y": 1.25},
                {"x": 7, "y": 1.25},
                {"x": 8, "y": 1.25},
                {"x": 9, "y": 1.25},
                {"x": 10, "y": 1.25},
                {"x": 11, "y": 1.25},
                {"x": 12, "y": 1.25},
                {"x": 13, "y": 1.25},
                {"x": 14, "y": 1.25},
                {"x": 15.25, "y": 1.25},
                {"x": 16.25, "y": 1.25},
                {"x": 17.25, "y": 1.25},

                {"x": 0, "y": 2.25, "w": 1.5},
                {"x": 1.5, "y": 2.25},
                {"x": 2.5, "y": 2.25},
                {"x": 3.5, "y": 2.25},
                {"x": 4.5, "y": 2.25},
                {"x": 5.5, "y": 2.25},
                {"x": 6.5, "y": 2.25},
                {"x": 7.5, "y": 2.25},
                {"x": 8.5, "y": 2.25},
                {"x": 9.5, "y": 2.25},
                {"x": 10.5, "y": 2.25},
                {"x": 11.5, "y": 2.25},
                {"x": 12.5, "y": 2.25},
                {"x": 13.5, "y": 2.25, "w": 1.5},
                {"x": 15.25, "y": 2.25},
                {"x": 16.25, "y": 2.25},
                {"x": 17.25, "y": 2.25},

                {"x": 0, "y": 3.25, "w": 1.75},
                {"x": 1.75, "y": 3.25},
                {"x": 2.75, "y": 3.25},
                {"x": 3.75, "y": 3.25},
                {"x": 4.75, "y": 3.25},
                {"x": 5.75, "y": 3.25},
                {"x": 6.75, "y": 3.25},
                {"x": 7.75, "y": 3.25},
                {"x": 8.75, "y": 3.25},
                {"x": 9.75, "y": 3.25},
                {"x": 10.75, "y": 3.25},
                {"x": 11.75, "y": 3.25},
                {"x": 12.75, "y": 3.25, "w": 2.25},

                {"x": 0, "y": 4.25, "w": 2.25},
                {"x": 2.25, "y": 4.25},
                {"x": 3.25, "y": 4.25},
                {"x": 4.25, "y": 4.25},
                {"x": 5.25, "y": 4.25},
                {"x": 6.25, "y": 4.25},
                {"x": 7.25, "y": 4.25},
                {"x": 8.25, "y": 4.25},
                {"x": 9.25, "y": 4.25},
                {"x": 10.25, "y": 4.25},
                {"x": 11.25, "y": 4.25},
                {"x": 12.25, "y": 4.25, "w": 1.75},
                {"x": 14, "y": 4.25},
                {"x": 16.25, "y": 4.25},

                {"x": 0, "y": 5.25, "w": 1.25},
                {"x": 1.25, "y": 5.25, "w": 1.25},
                {"x": 2.5, "y": 5.25, "w": 1.25},
                {"x": 3.75, "y": 5.25, "w": 6.25},
                {"x": 10, "y": 5.25, "w": 1.25},
                {"x": 11.25, "y": 5.25, "w": 1.25},
                {"x": 12.5, "y": 5.25, "w": 1.25},
                {"x": 13.75, "y": 5.25, "w": 1.25},
                {"x": 15.25, "y": 5.25},
                {"x": 16.25, "y": 5.25},
                {"x": 17.25, "y": 5.25}
            ]
        },
        "LAYOUT_tkl_ansi_split_bs_rshift_tsangan": {
            "layout": [
                {"x": 0, "y": 0},
                {"x": 2, "y": 0},
                {"x": 3, "y": 0},
                {"x": 4, "y": 0},
                {"x": 5, "y": 0},
                {"x": 6.5, "y": 0},
                {"x": 7.5, "y": 0},
                {"x": 8.5, "y": 0},
                {"x": 9.5, "y": 0},
                {"x": 11, "y": 0},
                {"x": 12, "y": 0},
                {"x": 13, "y": 0},
                {"x": 14, "y": 0},
                {"x": 15.25, "y": 0},
                {"x": 16.25, "y": 0},
                {"x": 17.25, "y": 0},

                {"x": 0, "y": 1.25},
                {"x": 1, "y": 1.25},
                {"x": 2, "y": 1.25},
                {"x": 3, "y": 1.25},
                {"x": 4, "y": 1.25},
                {"x": 5, "y": 1.25},
                {"x": 6, "y": 1.25},
                {"x": 7, "y": 1.25},
                {"x": 8, "y": 1.25},
                {"x": 9, "y": 1.25},
                {"x": 10, "y": 1.25},
                {"x": 11, "y": 1.25},
                {"x": 12, "y": 1.25},
                {"x": 13, "y": 1.25},
                {"x": 14, "y": 1.25},
                {"x": 15.25, "y": 1.25},
                {"x": 16.25, "y": 1.25},
                {"x": 17.25, "y": 1.25},

                {"x": 0, "y": 2.25, "w": 1.5},
                {"x": 1.5, "y": 2.25},
                {"x": 2.5, "y": 2.25},
                {"x": 3.5, "y": 2.25},
                {"x": 4.5, "y": 2.25},
                {"x": 5.5, "y": 2.25},
                {"x": 6.5, "y": 2.25},
                {"x": 7.5, "y": 2.25},
                {"x": 8.5, "y": 2.25},
                {"x": 9.5, "y": 2.25},
                {"x": 10.5, "y": 2.25},
                {"x": 11.5, "y": 2.25},
                {"x": 12.5, "y": 2.25},
                {"x": 13.5, "y": 2.25, "w": 1.5},
                {"x": 15.25, "y": 2.25},
                {"x": 16.25, "y": 2.25},
                {"x": 17.25, "y": 2.25},

                {"x": 0, "y": 3.25, "w": 1.75},
                {"x": 1.75, "y": 3.25},
                {"x": 2.75, "y": 3.25},
                {"x": 3.75, "y": 3.25},
                {"x": 4.75, "y": 3.25},
                {"x": 5.75, "y": 3.25},
                {"x": 6.75, "y": 3.25},
                {"x": 7.75, "y": 3.25},
                {"x": 8.75, "y": 3.25},
                {"x": 9.75, "y": 3.25},
                {"x": 10.75, "y": 3.25},
                {"x": 11.75, "y": 3.25},
                {"x": 12.75, "y": 3.25, "w": 2.25},

                {"x": 0, "y": 4.25, "w": 2.25},
                {"x": 2.25, "y": 4.25},
                {"x": 3.25, "y": 4.25},
                {"x": 4.25, "y": 4.25},
                {"x": 5.25, "y": 4.25},
                {"x": 6.25, "y": 4.25},
                {"x": 7.25, "y": 4.25},
                {"x": 8.25, "y": 4.25},
                {"x": 9.25, "y": 4.25},
                {"x": 10.25, "y": 4.25},
                {"x": 11.25, "y": 4.25},
                {"x": 12.25, "y": 4.25, "w": 1.75},
                {"x": 14, "y": 4.25},
                {"x": 16.25, "y": 4.25},

                {"x": 0, "y": 5.25, "w": 1.5},
                {"x": 1.5, "y": 5.25},
                {"x": 2.5, "y": 5.25, "w": 1.5},
                {"x": 4, "y": 5.25, "w": 7},
                {"x": 11, "y": 5.25, "w": 1.5},
                {"x": 12.5, "y": 5.25},
                {"x": 13.5, "y": 5.25, "w": 1.5},
                {"x": 15.25, "y": 5.25},
                {"x": 16.25, "y": 5.25},
                {"x": 17.25, "y": 5.25}
            ]
        },
        "LAYOUT_tkl_ansi_split_bs_rshift_symmetric": {
            "layout": [
                {"x": 0, "y": 0},
                {"x": 2, "y": 0},
                {"x": 3, "y": 0},
                {"x": 4, "y": 0},
                {"x": 5, "y": 0},
                {"x": 6.5, "y": 0},
                {"x": 7.5, "y": 0},
                {"x": 8.5, "y": 0},
                {"x": 9.5, "y": 0},
                {"x": 11, "y": 0},
                {"x": 12, "y": 0},
                {"x": 13, "y": 0},
                {"x": 14, "y": 0},
                {"x": 15.25, "y": 0},
                {"x": 16.25, "y": 0},
                {"x": 17.25, "y": 0},

                {"x": 0, "y": 1.25},
                {"x": 1, "y": 1.25},
                {"x": 2, "y": 1.25},
                {"x": 3, "y": 1.25},
                {"x": 4, "y": 1.25},
                {"x": 5, "y": 1.25},
                {"x": 6, "y": 1.25},
                {"x": 7, "y": 1.25},
                {"x": 8, "y": 1.25},
                {"x": 9, "y": 1.25},
                {"x": 10, "y": 1.25},
                {"x": 11, "y": 1.25},
                {"x": 12, "y": 1.25},
                {"x": 13, "y": 1.25},
                {"x": 14, "y": 1.25},
                {"x": 15.25, "y": 1.25},
                {"x": 16.25, "y": 1.25},
                {"x": 17.25, "y": 1.25},

                {"x": 0, "y": 2.25, "w": 1.5},
                {"x": 1.5, "y": 2.25},
                {"x": 2.5, "y": 2.25},
                {"x": 3.5, "y": 2.25},
                {"x": 4.5, "y": 2.25},
                {"x": 5.5, "y": 2.25},
                {"x": 6.5, "y": 2.25},
                {"x": 7.5, "y": 2.25},
                {"x": 8.5, "y": 2.25},
                {"x": 9.5, "y": 2.25},
                {"x": 10.5, "y": 2.25},
                {"x": 11.5, "y": 2.25},
                {"x": 12.5, "y": 2.25},
                {"x": 13.5, "y": 2.25, "w": 1.5},
                {"x": 15.25, "y": 2.25},
                {"x": 16.25, "y": 2.25},
                {"x": 17.25, "y": 2.25},

                {"x": 0, "y": 3.25, "w": 1.75},
                {"x": 1.75, "y": 3.25},
                {"x": 2.75, "y": 3.25},
                {"x": 3.75, "y": 3.25},
                {"x": 4.75, "y": 3.25},
                {"x": 5.75, "y": 3.25},
                {"x": 6.75, "y": 3.25},
                {"x": 7.75, "y": 3.25},
                {"x": 8.75, "y": 3.25},
                {"x": 9.75, "y": 3.25},
                {"x": 10.75, "y": 3.25},
                {"x": 11.75, "y": 3.25},
                {"x": 12.75, "y": 3.25, "w": 2.25},

                {"x": 0, "y": 4.25, "w": 2.25},
                {"x": 2.25, "y": 4.25},
                {"x": 3.25, "y": 4.25},
                {"x": 4.25, "y": 4.25},
                {"x": 5.25, "y": 4.25},
                {"x": 6.25, "y": 4.25},
                {"x": 7.25, "y": 4.25},
                {"x": 8.25, "y": 4.25},
                {"x": 9.25, "y": 4.25},
                {"x": 10.25, "y": 4.25},
                {"x": 11.25, "y": 4.25},
                {"x": 12.25, "y": 4.25, "w": 1.75},
                {"x": 14, "y": 4.25},
                {"x": 16.25, "y": 4.25},

                {"x": 0, "y": 5.25, "w": 1.25},
                {"x": 1.875, "y": 5.25, "w": 1.25},
                {"x": 3.125, "y": 5.25, "w": 1.25},
                {"x": 4.375, "y": 5.25, "w": 6.25},
                {"x": 10.625, "y": 5.25, "w": 1.25},
                {"x": 11.875, "y": 5.25, "w": 1.25},
                {"x": 13.755, "y": 5.25, "w": 1.25},
                {"x": 15.25, "y": 5.25},
                {"x": 16.25, "y": 5.25},
                {"x": 17.25, "y": 5.25}
            ]
        }
    }
}
```

File: keyboards/nightly_boards/n87/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    /* Base */
+    [0] = LAYOUT_standard(
+		KC_ESC,           KC_F1,   KC_F2,   KC_F3,   KC_F4,       KC_F5,   KC_F6,   KC_F7,   KC_F8,        KC_F9,   KC_F10,  KC_F11,  KC_F12,    KC_PSCR, KC_SLCK, KC_PAUS,
```
Comment (fauxpark) [inline keyboards/nightly_boards/n87/keymaps/default/keymap.c:21]:
```suggestion
    [0] = LAYOUT_tkl_ansi_split_bs_rshift(
        KC_ESC,           KC_F1,   KC_F2,   KC_F3,   KC_F4,       KC_F5,   KC_F6,   KC_F7,   KC_F8,        KC_F9,   KC_F10,  KC_F11,  KC_F12,    KC_PSCR, KC_SLCK, KC_PAUS,
```

File: keyboards/nightly_boards/n87/keymaps/default/keymap.c [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+        KC_GRV,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC, KC_BSPC,   KC_INS,  KC_HOME, KC_PGUP,
+        KC_TAB,      KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS,        KC_DEL,  KC_END,  KC_PGDN,
+        KC_CAPS,       KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_ENT,
+        KC_LSFT,            KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_LSFT,        MO(1),              KC_UP,
+        KC_LCTL,   KC_LGUI,   KC_LALT,                       KC_SPC,                              KC_LALT,   KC_LGUI,   KC_MENU,   KC_LCTL,      KC_LEFT, KC_DOWN, KC_RGHT
+    ),
+    [1] = LAYOUT_standard(
```
Comment (fauxpark) [inline keyboards/nightly_boards/n87/keymaps/default/keymap.c:28]:
```suggestion
    [1] = LAYOUT_tkl_ansi_split_bs_rshift(
```

File: keyboards/nightly_boards/n87/keymaps/symmetric_standard/keymap.c [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    /* Base */
+    [0] = LAYOUT_symmetric_standard(
+		KC_ESC,           KC_F1,   KC_F2,   KC_F3,   KC_F4,       KC_F5,   KC_F6,   KC_F7,   KC_F8,        KC_F9,   KC_F10,  KC_F11,  KC_F12,    KC_PSCR, KC_SLCK, KC_PAUS,
```
Comment (fauxpark) [inline keyboards/nightly_boards/n87/keymaps/symmetric_standard/keymap.c:21]:
```suggestion
    [0] = LAYOUT_tkl_ansi_split_bs_rshift_symmetric(
		KC_ESC,           KC_F1,   KC_F2,   KC_F3,   KC_F4,       KC_F5,   KC_F6,   KC_F7,   KC_F8,        KC_F9,   KC_F10,  KC_F11,  KC_F12,    KC_PSCR, KC_SLCK, KC_PAUS,
```

File: keyboards/nightly_boards/n87/keymaps/symmetric_standard/keymap.c [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+        KC_GRV,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC, KC_BSPC,   KC_INS,  KC_HOME, KC_PGUP,
+        KC_TAB,      KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS,        KC_DEL,  KC_END,  KC_PGDN,
+        KC_CAPS,       KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_ENT,
+        KC_LSFT,            KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_LSFT,        MO(1),              KC_UP,
+        KC_LCTL,   KC_LGUI,   KC_LALT,                       KC_SPC,                              KC_LALT,   KC_LGUI,   		     KC_LCTL,      KC_LEFT, KC_DOWN, KC_RGHT
+    ),
+    [1] = LAYOUT_symmetric_standard(
```
Comment (fauxpark) [inline keyboards/nightly_boards/n87/keymaps/symmetric_standard/keymap.c:28]:
```suggestion
    [1] = LAYOUT_tkl_ansi_split_bs_rshift_symmetric(
```

File: keyboards/nightly_boards/n87/keymaps/tsangan/keymap.c [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    /* Base */
+    [0] = LAYOUT_tsangan(
+		KC_ESC,           KC_F1,   KC_F2,   KC_F3,   KC_F4,       KC_F5,   KC_F6,   KC_F7,   KC_F8,        KC_F9,   KC_F10,  KC_F11,  KC_F12,    KC_PSCR, KC_SLCK, KC_PAUS,
```
Comment (fauxpark) [inline keyboards/nightly_boards/n87/keymaps/tsangan/keymap.c:21]:
```suggestion
    [0] = LAYOUT_tkl_ansi_split_bs_rshift_tsangan(
        KC_ESC,           KC_F1,   KC_F2,   KC_F3,   KC_F4,       KC_F5,   KC_F6,   KC_F7,   KC_F8,        KC_F9,   KC_F10,  KC_F11,  KC_F12,    KC_PSCR, KC_SLCK, KC_PAUS,
```

File: keyboards/nightly_boards/n87/keymaps/tsangan/keymap.c [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+        KC_GRV,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC, KC_BSPC,   KC_INS,  KC_HOME, KC_PGUP,
+        KC_TAB,      KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS,        KC_DEL,  KC_END,  KC_PGDN,
+        KC_CAPS,       KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_ENT,
+        KC_LSFT,            KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_LSFT,        MO(1),              KC_UP,
+        KC_LCTL,   KC_LGUI,   KC_LALT,                       KC_SPC,                                         KC_LALT,   KC_LGUI,   KC_LCTL,      KC_LEFT, KC_DOWN, KC_RGHT
+    ),
+    [1] = LAYOUT_tsangan(
```
Comment (fauxpark) [inline keyboards/nightly_boards/n87/keymaps/tsangan/keymap.c:28]:
```suggestion
    [1] = LAYOUT_tkl_ansi_split_bs_rshift_tsangan(
```

File: keyboards/nightly_boards/n87/n87.h [hunk 9]
```diff
@@ -0,0 +1,93 @@
...
+
+/* This is a shortcut to help you visually see your layout.
+ *
+ * The first section contains all of the arguments representing the physical
+ * layout of the board and position of the keys.
+ *
+ * The second converts the arguments into a two-dimensional array which
+ * represents the switch matrix.
+ */
+#define LAYOUT_standard( \
```
Comment (fauxpark) [inline keyboards/nightly_boards/n87/n87.h:29]:
```suggestion
#define LAYOUT_tkl_ansi_split_bs_rshift( \
```

File: keyboards/nightly_boards/n87/n87.h [hunk 10]
```diff
@@ -0,0 +1,93 @@
...
+    { K50,   K51,   K52,   K53,   K54,   K55,   K56,   K57,   K58   }, \
+    { K60,   K61,   K62,   K63,   K64,   K65,   K66,   KC_NO, KC_NO }, \
+    { KC_NO, K71,   K72,   K73,   K74,   K75,   K76,   KC_NO, KC_NO }, \
+    { K80,   K81,   K82,   K83,   K84,   K85,   K86,   KC_NO, K88   }, \
+    { KC_NO, K91,   K92,   K93,   K94,   K95,   K96,   KC_NO, KC_NO }, \
+    { KA0,   KA1,   KC_NO, KA3,   KC_NO, KA5,   KA6,   KC_NO, KA8   }, \
+    { KB0,   KC_NO, KC_NO, KC_NO, KC_NO, KB5,   KB6,   KB7,   KB8   }, \
+}
+
+#define LAYOUT_tsangan( \
```
Comment (fauxpark) [inline keyboards/nightly_boards/n87/n87.h:51]:
```suggestion
#define LAYOUT_tkl_ansi_split_bs_rshift_tsangan( \
```

File: keyboards/nightly_boards/n87/n87.h [hunk 11]
```diff
@@ -0,0 +1,93 @@
...
+    { K50,   K51,   K52,   K53,   K54,   K55,   K56,   K57,   K58   }, \
+    { K60,   K61,   K62,   K63,   K64,   K65,   K66,   KC_NO, KC_NO }, \
+    { KC_NO, K71,   K72,   K73,   K74,   K75,   K76,   KC_NO, KC_NO }, \
+    { K80,   K81,   K82,   K83,   K84,   K85,   K86,   KC_NO, K88   }, \
+    { KC_NO, K91,   K92,   K93,   K94,   K95,   K96,   KC_NO, KC_NO }, \
+    { KA0,   KA1,   KC_NO, KA3,   KC_NO, KC_NO,   KA6,   KC_NO, KA8   }, \
+    { KB0,   KC_NO, KC_NO, KC_NO, KC_NO, KB5,   KB6,   KB7,   KB8   }, \
+}
+
+#define LAYOUT_symmetric_standard( \
```
Comment (fauxpark) [inline keyboards/nightly_boards/n87/n87.h:73]:
```suggestion
#define LAYOUT_tkl_ansi_split_bs_rshift_symmetric( \
```

File: keyboards/nightly_boards/n87/info.json [hunk 12]
```diff
@@ -0,0 +1,287 @@
...
+                {"x": 11.875, "y": 5.25, "w": 1.25},
+                {"x": 13.755, "y": 5.25, "w": 1.25},
+                {"x": 15.25, "y": 5.25},
+                {"x": 16.25, "y": 5.25},
+                {"x": 17.25, "y": 5.25}
+            ]
+        }
+    }
+}
+}
```
Comment (Erovia) [inline keyboards/nightly_boards/n87/info.json:287]:
```suggestion

```

File: keyboards/nightly_boards/n87/rules.mk [hunk 13]
```diff
@@ -0,0 +1,32 @@
...
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
+BOOTLOADER = atmel-dfu
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration
```
[the remark this answers is not in the document]
Comment (NightlyBoards) [reply keyboards/nightly_boards/n87/rules.mk]: Edit done.

File: keyboards/nightly_boards/n87/readme.md [hunk 14]
```diff
@@ -0,0 +1,12 @@
+# [n]87
+
```
[the remark this answers is not in the document]
Comment (NightlyBoards) [reply keyboards/nightly_boards/n87/readme.md:2]: Added an imgur link of the board

File: keyboards/nightly_boards/n87/info.json (hunk 12 above)
Comment (NightlyBoards) [reply keyboards/nightly_boards/n87/info.json:286]: edited

**Comment by NightlyBoards** [comment]
Latest commit also have updated keymaps to enable/adjust audio click

File: keyboards/nightly_boards/n87/n87.h (hunk 11 above)
Comment (NightlyBoards) [reply keyboards/nightly_boards/n87/n87.h]: https://github.com/qmk/qmk_firmware/pull/9663/commits/288683f30bbe7d6903ea87e3c779db7f02e0921c

File: keyboards/nightly_boards/n87/n87.h (hunk 10 above)
Comment (NightlyBoards) [reply keyboards/nightly_boards/n87/n87.h]: https://github.com/qmk/qmk_firmware/pull/9663/commits/288683f30bbe7d6903ea87e3c779db7f02e0921c

File: keyboards/nightly_boards/n87/n87.h (hunk 9 above)
Comment (NightlyBoards) [reply keyboards/nightly_boards/n87/n87.h]: https://github.com/qmk/qmk_firmware/pull/9663/commits/288683f30bbe7d6903ea87e3c779db7f02e0921c

File: keyboards/nightly_boards/n87/keymaps/tsangan/keymap.c (hunk 8 above)
Comment (NightlyBoards) [reply keyboards/nightly_boards/n87/keymaps/tsangan/keymap.c]: https://github.com/qmk/qmk_firmware/pull/9663/commits/288683f30bbe7d6903ea87e3c779db7f02e0921c

File: keyboards/nightly_boards/n87/keymaps/tsangan/keymap.c (hunk 7 above)
Comment (NightlyBoards) [reply keyboards/nightly_boards/n87/keymaps/tsangan/keymap.c]: https://github.com/qmk/qmk_firmware/pull/9663/commits/288683f30bbe7d6903ea87e3c779db7f02e0921c

File: keyboards/nightly_boards/n87/keymaps/symmetric_standard/keymap.c (hunk 6 above)
Comment (NightlyBoards) [reply keyboards/nightly_boards/n87/keymaps/symmetric_standard/keymap.c]: https://github.com/qmk/qmk_firmware/pull/9663/commits/288683f30bbe7d6903ea87e3c779db7f02e0921c

File: keyboards/nightly_boards/n87/keymaps/symmetric_standard/keymap.c (hunk 5 above)
Comment (NightlyBoards) [reply keyboards/nightly_boards/n87/keymaps/symmetric_standard/keymap.c]: https://github.com/qmk/qmk_firmware/pull/9663/commits/288683f30bbe7d6903ea87e3c779db7f02e0921c

File: keyboards/nightly_boards/n87/keymaps/default/keymap.c (hunk 4 above)
Comment (NightlyBoards) [reply keyboards/nightly_boards/n87/keymaps/default/keymap.c]: https://github.com/qmk/qmk_firmware/pull/9663/commits/288683f30bbe7d6903ea87e3c779db7f02e0921c

File: keyboards/nightly_boards/n87/keymaps/default/keymap.c (hunk 3 above)
Comment (NightlyBoards) [reply keyboards/nightly_boards/n87/keymaps/default/keymap.c]: https://github.com/qmk/qmk_firmware/pull/9663/commits/288683f30bbe7d6903ea87e3c779db7f02e0921c

File: keyboards/nightly_boards/n87/info.json (hunk 2 above)
Comment (NightlyBoards) [reply keyboards/nightly_boards/n87/info.json]: https://github.com/qmk/qmk_firmware/pull/9663/commits/288683f30bbe7d6903ea87e3c779db7f02e0921c

File: keyboards/nightly_boards/n87/info.json (hunk 12 above)
Comment (NightlyBoards) [reply keyboards/nightly_boards/n87/info.json:286]: https://github.com/qmk/qmk_firmware/pull/9663/commits/916a7f49a282717c92986d447740e3bc6f97a829

File: keyboards/nightly_boards/n87/readme.md (hunk 14 above)
[the remark this answers is not in the document]
Comment (NightlyBoards) [reply keyboards/nightly_boards/n87/readme.md:2]: https://github.com/qmk/qmk_firmware/pull/9663/commits/916a7f49a282717c92986d447740e3bc6f97a829

File: keyboards/nightly_boards/n87/rules.mk (hunk 13 above)
[the remark this answers is not in the document]
Comment (NightlyBoards) [reply keyboards/nightly_boards/n87/rules.mk]: https://github.com/qmk/qmk_firmware/pull/9663/commits/916a7f49a282717c92986d447740e3bc6f97a829?file-filters%5B%5D=.c&file-filters%5B%5D=.h&file-filters%5B%5D=.json&file-filters%5B%5D=.md

**Comment by drashna** [comment]
Thanks!

**Pull request merged**